deploy/infrastructure/dev/us-east-2/cloudfront.tf

```diff
@@ -13,14 +13,14 @@ resource "aws_cloudfront_distribution" "cdn" {
   ]
   price_class = "PriceClass_All"
 
-  # The node named `ber` in dev environment uses an identity that is whitelisted by Lotus 
+  # The node named `assigner` in dev environment uses an identity that is whitelisted by Lotus 
   # bootstrap nodes in order to relay gossipsub. That node is also configured to re-propagate 
   # HTTP announces over gossipsub.
   # Therefore, all HTTP announce requests are routed to it. 
   # 
-  # See: storetheindex/ber-indexer ingress object.
+  # See: storetheindex/assigner-indexer ingress object.
   origin {
-    domain_name = "ber.${aws_route53_zone.dev_external.name}"
+    domain_name = "assigner.${aws_route53_zone.dev_external.name}"
```

Comment:
This needs to be manually applied by running terraform commands. See runbook on how to do this before merge

Reply:
done